dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/PreCheckDataRelationsValidationHook.java

```diff
@@ -419,29 +414,43 @@ private Program getEnrollmentProgramFromEvent( TrackerBundle bundle, Event event
         return null;
     }
 
-    private String getTrackedEntityTypeUidFromEnrollment( TrackerBundle bundle,
+    private void validateTrackedEntityTypeMatchesPrograms( ValidationErrorReporter reporter, Program program,
```

Comment:
It's supposed to read like `program's` since we are comparing if the enrollments tracked entity type matches the program's te type.

I can change it if that's not coming across. I tried to use the shortest meaningful name :joy: